lib/freeze.js

```diff
@@ -57,7 +51,8 @@ module.exports = class Freeze {
     }));
 
     this.github.issues.createComment(context.issue({
-      body: 'Sure thing. I\'ll close this issue for a bit. I\'ll ping you around ' + props.unfreezeMoment.calendar() + ' :clock1: ' +
+      body: 'Sure thing. I\'ll close this issue for a bit. I\'ll ping you around ' +
+      moment(props.unfreezeMoment).calendar() + ' :clock1: ' +
```

Comment:
This is actually causing the tests to fail now because `moment().calendar()` returns the date as `mm/dd/yyyy`. I actually can't figure out how the tests were even passing before.

Reply:
Nevermind. I just came back to this to try to fix it and the tests are actually passing. 😕

Reply:
:+1:. I'll run all this through again on master once the merges happen to make sure things are clean.

At the end of the day, we are 🔥 much of this to go to metadata anyway, so that should help